Jenkinsfile

```diff
@@ -0,0 +1,26 @@
+pipeline {
```

Comment:
Should have SPDX headers (if you want to check this in - though note we don't use Jenkins in the open source build, though no issue having in)